release/build_release.sh

```diff
@@ -68,6 +68,47 @@ function check_usage {
 	LIFERAY_COMMON_LOG_DIR="${_BUILD_DIR}"
 }
 
+function cherry_pick_commits {
+	local liferay_release_tickets_array
+
+	IFS=',' read -ra liferay_release_tickets_array <<< "${LIFERAY_RELEASE_TICKETS}"
+
+	lc_cd "${_PROJECTS_DIR}"/liferay-portal-ee
+
+	git pull origin -q "${LIFERAY_RELEASE_GIT_REF}"
+
+	local push_to_origin="true"
+
+	for liferay_release_ticket in "${liferay_release_tickets_array[@]}"
+	do
+		git checkout -q master
+
+		local liferay_release_commits_sha
+
+		IFS=$'\n' read -r -d '' -a liferay_release_commits_sha < <(git log --grep="${liferay_release_ticket}" --pretty=format:%H --reverse)
```

Comment:
@kiwm, do we need the `$` in `IFS=$'\n'`? If not, please remove

In `liferay_release_commits_sha < <(git log `, do we need the blank space between the characters `<`?

Reply:
The IFS=$'\n' is not need in this case but the `liferay_release_commits_sha < <(git log`  syntax is need.